pkg/csi_driver/multishare_controller.go

```diff
@@ -57,16 +57,17 @@ const (
 
 // MultishareController handles CSI calls for volumes which use Filestore multishare instances.
 type MultishareController struct {
-	driver                     *GCFSDriver
-	fileService                file.Service
-	cloud                      *cloud.Cloud
-	opsManager                 *MultishareOpsManager
-	volumeLocks                *util.VolumeLocks
-	ecfsDescription            string
-	isRegional                 bool
-	clustername                string
-	featureMaxSharePerInstance bool
-	featureMultishareBackups   bool
+	driver                          *GCFSDriver
+	fileService                     file.Service
+	cloud                           *cloud.Cloud
+	opsManager                      *MultishareOpsManager
+	volumeLocks                     *util.VolumeLocks
+	ecfsDescription                 string
+	isRegional                      bool
+	clustername                     string
+	featureMaxSharePerInstance      bool
+	featureMultishareBackups        bool
+	FeatureNFSExportOptionsOnCreate bool
```

Comment:
not sure why `featureMaxSharePerInstance` and `featureMultishareBackups` are lower cases here but consider align with them and use lower case

Reply:
Good catch. Fixed

Reply:
I don't see it reflected for some reason, can you double check

Reply:
Oops. Now actually fixed